tuf/api/metadata.py

```diff
@@ -1258,17 +1259,99 @@ def to_dict(self) -> Dict[str, Any]:
         return snapshot_dict
 
 
+class SuccinctHashDelegations:
+    """A container with information about a succinct hash delegation.
```

Comment:
I think this needs an explanation: "succinct hash delegation" is probably word salad for anyone except about 20 people on the planet.

It does not need to explain every detail but the core idea: that it desribes delegations of all possible target paths to variable number of bins, the names of which are generated based on the two fields.